yarpnarp.go

```diff
@@ -80,10 +104,26 @@ func RunYarpNarp() {
 
 	sort.Sort(zusagen)
 
+	var width int
+	var winsize struct {
+		wsRow    uint16
+		wsCol    uint16
+		wsXpixel uint16
+		wsYpixel uint16
+	}
+
+	if _, _, errno := unix.Syscall(unix.SYS_IOCTL, os.Stdout.Fd(), unix.TIOCGWINSZ, uintptr(unsafe.Pointer(&winsize))); errno != 0 {
+		width = 0
```

Comment:
Ich würde wenn dann < 0 benutzen zum signalisieren von "nicht truncaten". Negative Werte sind klar ungültig als Länge. Potentiell wäre aber out-of-band signalling eh klarer (i.e. ne zusätzliche Variable übergeben für "das soll truncated werden"). Siehe auch oben.